Tests/Interactive/Common/GameDebug.cs

```diff
@@ -0,0 +1,52 @@
+// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System.Collections.Generic;
+
+namespace MonoGame.InteractiveTests
+{
+    /// <summary>
+    /// Allows tests to output console messages including spammy messages that
+    /// may be throttled.
+    ///
+    /// On various platforms, this may be available via console output or via
+    /// a special console viewer (Console app on Mac; `adb logcat` on Android etc).
+    /// </summary>
+    public partial class GameDebug
+    {
+        /// <summary>Output a single console message.</summary>
+        public static void C(string message)
+        {
+            System.Console.WriteLine($"MGDBG: {message}");
+        }
+
+        /// <summary>Output an error message to the console.</summary>
+        public static void E(string message)
```

Comment:
I think I mentioned this on the other PR. 
It might be better to have actual descriptive method names like `LogError` or `Error`. Same for the `C` method. Looking at the code which is calling it , it is not obvious what the method does `GameDebug.C ("foo")` could be doing anything, so you have to go look. 

In this case `C` might be better being called `Log` or `Debug`.

Reply:
I was going for the minimal [Android-y](https://developer.android.com/reference/android/util/Log)  `Log` that are concise and help avoid cognitive overload when reading a bunch of the logging statements.

If the xmldoc comment doesn't help (on hover for example), I am okay with changing it to be a bit longer like you suggest.  But I wanted to provide you a rationale first.

It also helps with `using GD=GraphicsDebug; GD.E("...");` etc which makes it super concise.

Reply:
I understand the rationale. I would favour consistencey though, most of the logging in the repo uses things like `Debug.WriteLine` or `Log.Verbose` etc (see https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Platform/Android/MonoGameAndroidGameView.cs#L778). Keeping code consitent with other uses within the repo makes it easier to people to understand the code as a whole. This is my preference, we can see what other @MonoGame/maintainers think.

Reply:
I would agree with @dellis1972 , as this is the standard used by the rest of MonoGame.

Reply:
Done PTAL, thanks!